src/main/java/org/yunhwan/moviereview/service/MovieServiceImpl.java

```diff
@@ -124,10 +105,13 @@ public MovieDTO getMovie(Long mno) {
     @Transactional
     @Override
     public void modify(MovieDTO movieDTO) {
-        Movie movie = movieRepository.getOne(movieDTO.getMno());
-        log.info(movieDTO);
-        movie.changeTitle(movieDTO.getTitle(), movieDTO.getOpenDate(), movieDTO.getRunningTime()
-                , movieDTO.getCountry(), movieDTO.getRating());
-        // 엔티티매니저가 "변경 감지" 하기 때문에 따로 save 쿼리 안날렷음.
+        log.info("영화 수정 정보 movieDTO : {}", movieDTO);
+        Movie movie = movieRepository.findById(movieDTO.getMno())
+                .orElseThrow(() -> new IllegalArgumentException("해당 영화는 존재하지 않습니다."));
+
+        movie.changeTitle(movieDTO.getTitle(), movieDTO.getOpenDate(),
+                movieDTO.getRunningTime(), movieDTO.getCountry(),
+                movieDTO.getRating());
```

Comment:
```suggestion
        movie.modify(
                movieDTO.getTitle(), 
                movieDTO.getOpenDate(),
                movieDTO.getRunningTime(), 
                movieDTO.getCountry(),
                movieDTO.getRating()
        );
```
- title만 변경하는건 아니군요~
- 변경시 유효성 검증은 필요 없을까요? (title 제약사항, openDate 범위, rating 정책, country 검증 등)